README.md

```diff
@@ -1,15 +1,111 @@
-# Sourceloop Angular Boiler Plate
+#### ANGULAR BOILERPLATE
```

Comment:
Make it ARC Angular Boilerplate. and use h1 heading (using a single #).

Reply:
done